docs/plugins/lyrics.rst

```diff
@@ -129,11 +129,6 @@ few suggestions.
 Activate Google Custom Search
 ------------------------------
 
-Using the Google backend requires `BeautifulSoup`_, which you can install
-using `pip`_ by typing::
-
-    pip install beautifulsoup4
-
```

Comment:
I think I would actually like to leave this copy of the text here? Because that way the section for the Google backend can stand alone. (Otherwise, it might be easy to miss that there's another section.)